components/graphs.py

```diff
@@ -0,0 +1,30 @@
+import pandas as pd
+import plotly.express as px
+
+
+def make_map(df):
+    '''
+    Generates interactive graph of Species and Subspecies by location.
+    
+    Parameters:
+    -----------
+    df - dataframe of specimens
+
+    Returns: 
+    --------
+    fig - Map of their locations
```

Comment:
Is it clear what type the returned "Map" object is going to be, or would it be worth saying?

Reply:
It's just a Plotly figure object, so I think the variable name `fig` is sufficient (and reasonably standard).